lib/pages/settings/view/settings_page.dart

```diff
@@ -33,6 +47,8 @@ class _SettingsPageState extends State<SettingsPage> {
     Provider.of<AuthProvider>(context, listen: false)
         .isVerified
         .then((value) => setState(() => isVerified = value));
+    // Initialisation of Package information
```

Comment:
Unnecessary comment --  I can tell that from the method name, just leave an empty line here.